src/Props/faq/faq.ts

```diff
@@ -0,0 +1,69 @@
+  },
+  {
+    question: "How long is the event?",
+    answer:
+      "CruzHacks is a 3 day weekend event from Friday night to Sunday noon with an options; awards ceremony in the afternoon.",
+  },
+  {
+    question: "Who can participate?",
+    answer:
+      "Anyone and everyone! It is open to anyone who wants to create their vision. No coding or technical experience is necessary. If you are a designer, product manager, artists, business major, etc you are welcome to apply! Mentors will be available to help you through any technical difficulties.",
+  },
+  {
+    question:
+      "Do I need to have a team in place/at all? What would be the limit if so?",
+    answer:
+      "It is not necessary to have a team at the time of application nor at the actual event. Teams must be a max of 4 people.",
+  },
+  {
+    question: "Do I need to know how to code?",
+    answer:
+      "Nope! No experience is required. We’re looking for applicants who are passionate about learning something new and creating a vision. 25% of hackers in 2021 and 47% in 2020 were first time hackers.",
+  },
+  {
+    question: "Do I need to be a Computer Science/Engineering major?",
+    answer: "We welcome and accept people of all majors!",
+  },
+  {
+    question: "Will CruzHacks be virtual or in-person?",
+    answer:
+      "Due to UCSC, Santa Cruz, and California protocols, CruzHacks will remain virtual for CruzHacks 2022. However, there may be a possibility of having a short in-person event for those in the Santa Cruz county that abide by UCSC protocols and feel comfortable to attend.",
+  },
+  {
+    question: "Does it cost anything to participate",
+    answer: "Cruzhacks is free for all participants!",
+  },
+  {
+    question: "What kind of projects can I make?",
+    answer:
+      "Any project (as technical as you want to get) is acceptable as long as it is part of one of the three main categories: Earth, Health, or Social Justice.",
+  },
+  {
+    question: "Do I have to be a UCSC student",
+    answer:
+      "Nope! We accept applicants from all over the world. In 2021, 47% of our hackers were non-UCSC students and among that, there were about 100 different schools.",
+  },
+  {
+    question: "Will there be people there to help me with my project?",
+    answer:
+      "Yes! We have at least 25 mentors ready to assist anyone on their project varying all subjects and all skill levels.",
+  },
+  {
+    question: "Will there be any workshops? On what topics?",
+    answer:
+      "We have a wide range of workshops, both technical and non-technical, planned including but not limited to: Intro to Web Design, Basics of HTML, Machine learning basics, and Resume/Cover Letter.",
+  },
+  {
```

Comment:
Could we try fixing the vertical spacing on the third column to match the design a little bit more.